packages/js/components/src/tooltip/tooltip.tsx

```diff
@@ -71,14 +71,13 @@ export const Tooltip: React.FC< TooltipProps > = ( {
 
 				{ isPopoverVisible && (
 					<Popover
-						focusOnMount="container"
+						focusOnMount="firstElement"
```

Comment:
`true` is for container: https://github.com/WordPress/gutenberg/blob/ca2ee4d6fa8c7044cd1d3045c0985a77659f44fa/packages/components/src/popover/README.md#L142